2016-01-06/swagger/Lifecycle.json

```diff
@@ -0,0 +1,92 @@
+{
```

Comment:
File should be named like `bucket_policy.json`, for lifecycle, it should be `bucket_lifecycle.json`